chingu-coders/Voyage2-Turtles-02
# newsfeed prototype, github pages deployment bugfix, some JS refactoring

## Purpose
* Implements basic newsfeed component
* Fixes a bug that was preventing proper loading of bg images and sounds in github pages demo
* Screwing around with the MVC structure

## Approach/Fixes
* Newsfeed is a core feature of the app. Its not pretty right now, but it works. Could def use some help with styling it.
* The above-mentioned bug was the result of improperly referencing of relative paths for assets
* I started messing with the MVC structure because I've never worked on an app this complex, and I am a bit unsure of how each component should be organized. its a work in progress...

## Testing
* Check out the pull request
* Click the news icon and confirm that main newsfeed appears
* confirm that there are three articles each from three sources: Hacker news, tech crunch, and recode (we can change the sources later if we want. I am not so sure that recode fits our theme)
* note that some images will not load, while others will be replaced with the turtle icon. the turtle icon is deliberate: its used in place of articles that dont have an image. The articles with no image at all are actually returning a url for an image, but for some reason it wont display. Its a bug and I am going to work on it.
